api/env.template

```diff
@@ -4,11 +4,6 @@ DJANGO_SETTINGS_MODULE=conf.settings
 DJANGO_SECRET_KEY="ny#b__$$f6ry4wy8oxre97&-68u_0lk3gw(z=d40_dxey3zw0v1"
 DJANGO_DEBUG_ENABLED=True
 
-BASE_URL=http://localhost:50280/
-ENVIRONMENT=development
-# List of comma-separated hosts/domain names, e.g., 127.17.0.1,local.app
-ALLOWED_HOSTS=localhost,172.17.0.1,host.docker.internal
```

Comment:
No need to duplicate these between `env.docker`. These environment files are now exclusively used for local development, so we don't need to do anything to make sure it's easy to handle live vs local deployments of it.